ci/pyinstaller/hook-watchmaker.py

```diff
@@ -6,7 +6,13 @@
 from PyInstaller.utils.hooks import (collect_data_files, collect_dynamic_libs,
                                      collect_submodules, copy_metadata)
 
-datas = []
+datas = [
+    ("src/watchmaker/static", "watchmaker/static"),
```

Comment:
Minor nit.  Can you match the formatting for dirs?

Reply:
This is how `black` formats the file, I don't think we should fight with the tool.

Reply:
That's interesting.  Okay, it's good with me.